src/core_write.cpp

```diff
@@ -41,7 +41,7 @@ std::string FormatScript(const CScript& script)
                 ret += strprintf("%i ", op - OP_1NEGATE - 1);
                 continue;
             } else if (op >= OP_NOP && op <= OP_NOP10) {
-                std::string str(GetOpName(op));
+                std::string str = GetOpName(op);
```

Comment:
Why this change is needed?

Reply:
I don't think it's needed. I'll revert the change.